src/browserlib/extract-cssdfn.mjs

```diff
@@ -103,7 +148,11 @@ const extractDfns = (doc, selector, extractor, { unique } = { unique: true }) =>
     .forEach(dfn => {
       if (res[dfn.name]) {
         if (unique) {
-          throw new Error(`More than one dfn found for CSS property/descriptor "${dfn.name}"`);
+          const merged = mergeDfns(res[dfn.name], dfn);
+          if (!merged) {
+            throw new Error(`More than one dfn found for CSS property "${dfn.name}" and dfns cannot be merged`);
```

Comment:
just to be sure, we don't expect any current spec to throw, right?

Reply:
The update is motivated by two current specs for which previous code throws. Once that is merged, all current specs should pass.